lib/KeyboardAwareMixin.js

```diff
@@ -112,7 +114,8 @@ const KeyboardAwareMixin = {
   /**
    * @param extraHeight: takes an extra height in consideration.
    */
-  scrollToFocusedInput: function (reactNode: Object, extraHeight: number = this.props.extraHeight) {
+  scrollToFocusedInput: function (reactNode: Object, extraHeight: number) {
```

Comment:
Flow was unable to understand what `this` represented at this point in time.
Looking at the release notes for flow 0.40.0 https://github.com/facebook/flow/releases/tag/v0.40.0 it may have been fixed a few days ago. So I'll double check it